crates/ruff_python_formatter/tests/snapshots/black_compatibility@py_310__pattern_matching_simple.py.snap

```diff
@@ -133,13 +133,13 @@ def where_is(point):
  
  match command.split():
 -    case ["go", ("north" | "south" | "east" | "west")]:
-+    case ["go", NOT_YET_IMPLEMENTED_PatternMatchOf | (y)]:
++    case ["go", (NOT_YET_IMPLEMENTED_PatternMatchOf | (y))]:
```

Comment:
Regarding the addition of parentheses around unions, I believe that will be handled in `MatchOr`?